app/test_main.py

```diff
@@ -1,3 +1,24 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected_result",
+    [
+        (0, 0, [0, 0]),
+        (15, 15, [1, 1]),
+        (28, 28, [3, 2]),
+        (28, 29, [3, 3]),
+        (168, 164, [38, 30]),
+    ],
+)
```

Comment:
Read the checklist, and check which cases you haven't checked. Also you should check all age boundaries